client/my-sites/email/inbox/mailbox-selection-list/style.scss

```
@@ -0,0 +1,196 @@
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__new-mailbox-upsell-container {
+		background: #f9f9f9;
+		align-items: center;
+		justify-content: space-between;
+		position: relative;
+		margin: 20px auto 10px;
+		padding: 10px 15px;
+		width: 240px;
+
+		@include break-medium {
+			width: 342px;
+			display: flex;
+			margin-top: 60px;
+			padding: 16px 24px;
+		}
+
+		@include break-large {
+			width: 660px;
+			display: flex;
+			margin-top: 80px;
+			padding: 16px 24px;
+		}
+
+		.mailbox-selection-list__new-mailbox-upsell-messages {
+			font-size: $font-body-small;
+			line-height: 28px;
+
+			h2 {
+				font-weight: bold;
+				font-size: $font-body;
+				line-height: 30px;
+				margin-bottom: 0;
+
+				@include break-medium {
+					margin-bottom: 8px;
+					line-height: 34px;
+				}
+
+				@include break-large {
+					margin-bottom: 8px;
+					line-height: 34px;
+				}
+			}
+		}
+
+		.mailbox-selection-list__new-mailbox-upsell-cta {
+			margin-top: 20px;
+
+			@include break-medium {
+				margin-top: 0;
+			}
+
+			@include break-large {
+				margin-top: 0;
+			}
+
+			.button {
+				font-size: $font-body-small;
+				font-weight: bold;
+			}
+		}
+	}
+
+	.mailbox-selection-list__status {
+		line-height: 24px;
+		font-size: $font-title-small;
+		position: fixed;
+		top: 50%;
+		left: 50%;
+		transform: translate( -50%, -50% );
+		display: flex;
+		align-items: center;
+		justify-content: center;
+
+		.gridicon {
+			margin-right: 14px;
+			width: 40px;
+			height: 40px;
+		}
+
+		.gridicons-cross-circle {
+			color: #ff0000;
+		}
+
+		@include break-large {
+			transform: none;
+			font-size: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__mailbox {
+		width: 310px;
+		box-shadow: none;
+		border-width: 1px 0 0;
+		border-color: #e7e9ea;
+		border-style: solid;
+		margin-bottom: 1px;
+		display: flex;
+		align-items: center;
+		padding: 12px;
+
+		@include break-medium {
+			width: 343px;
+			padding: 16px;
+		}
+
+		@include break-large {
+			width: 660px;
+		}
+
+		.gridicons-external {
+			width: 20px;
+		}
+
+		&:first-of-type {
+			border-top-width: 0;
+
+			@include break-large {
+				margin-top: 60px;
+			}
+		}
+
+		.mailbox-selection-list__domain h2 {
+			color: var( --p2-color-text );
+			font-family: $sans;
+			font-weight: 600;
+			font-size: $font-body;
+			word-break: break-all;
+		}
+	}
+
+	.mailbox-selection-list__domain {
+		margin: 0;
+		padding: 0;
+	}
+
+	.mailbox-selection-list__icon {
+		align-self: center;
+		height: 36px;
+		width: 36px;
+		margin-right: 16px;
+
+		.mailbox-selection-list__icon-circle {
+			width: 100%;
+			height: 100%;
+			line-height: 100%;
+			border-radius: 999px; /* stylelint-disable-line scales/radii */
+			font-size: $font-title-small;
+			color: var( --studio-white );
+			text-align: center;
+			background: #3197a7;
+			font-weight: bold;
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			text-transform: capitalize;
+		}
```

General comment/nit: I find it much easier to find specific CSS values when they're in alphabetical order. Without ordering, finding values takes more time for me. This applies across the whole file.

Also, does it make more sense to use `background-color` instead of just `background`?

And what impact does `line-height: 100%` have?
```suggestion
		.mailbox-selection-list__icon-circle {
			align-items: center;
			background: #3197a7;
			border-radius: 999px; /* stylelint-disable-line scales/radii */
			color: var( --studio-white );
			display: flex;
			font-size: $font-title-small;
			font-weight: bold;
			height: 100%;
			justify-content: center;
			line-height: 100%;
			text-align: center;
			text-transform: capitalize;
			width: 100%;
		}
```

---

For the `background`, in this case, the specific version is a matter of preference since the results are the same - but shorter strokes.
The `line-height` is redundant because it's always going to be one line of text - a character.